src/test/java/org/killbill/billing/plugin/dwolla/EmbeddedDbHelper.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2015 Groupon, Inc
+ *
+ * Groupon licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.plugin.dwolla;
+
+import org.killbill.billing.plugin.TestUtils;
+import org.killbill.billing.plugin.dwolla.dao.DwollaDao;
+import org.killbill.commons.embeddeddb.mysql.MySQLStandaloneDB;
+
+public class EmbeddedDbHelper {
```

Comment:
This (and other) utility classes should be in the [base framework](https://github.com/killbill/killbill-plugin-framework-java/blob/work-for-release-0.17.0/src/test/java/org/killbill/billing/plugin/TestWithEmbeddedDBBase.java) now (Adyen was just never updated).